app/src/components/TokenCard.tsx

```diff
@@ -0,0 +1,23 @@
+
+import React from 'react';
+
+import { Box, Direction, Image, PaddingSize, Spacing, Stack, Text, TextAlignment } from '@kibalabs/ui-react';
+
+export const TokenCard = (): React.ReactElement => {
+  return (
+    <Stack direction={Direction.Vertical} isFullHeight={true} isFullWidth={true} padding={PaddingSize.Wide}>
+      <Text variant='header3-singleLine'>Your Holdings</Text>
+      <Spacing variant={PaddingSize.Wide} />
+      <Box variant='card' height='204px' width='140px'>
+        <Box height='140px' width='140px'>
+          <Image source={'/assets/icon.png'} alternativeText='image' fitType='contain' />
+        </Box>
```

Comment:
why not just use a CollectionToken from the start, this seems so far behind cos its not connected up